lib/app_web/views/item_view.ex

```diff
@@ -1,3 +1,7 @@
 defmodule AppWeb.ItemView do
   use AppWeb, :view
+
+  def render("index.json", data) do
+    Enum.map(data.items, fn i -> %{id: i.id, text: i.text} end)
```

Comment:
I've simplified the structure to just returns a json with id and text.
We can't return directly the Item schema as some fields can't be converted automatically to json values.
One option would have been to use Json.Encoder on the schema (https://stackoverflow.com/questions/32549712/encoding-a-ecto-model-to-json-in-elixir) however it doesn't seem to be the best practice and building the json on view seems to be the way to go: https://elixirforum.com/t/any-alternatives-for-derive-and-poison-encoder/15002/2